addons/l10n_tr_nilvera_einvoice_extended/views/product_views.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <!-- view used for product.template only -->
+        <record id="product_template_only_form_view_inherit_l10n_tr_nilvera_extended" model="ir.ui.view">
+            <field name="name">product.template.product.form.view.inherit.l10n.tr.nilvera.extended</field>
+            <field name="model">product.template</field>
+            <field name="inherit_id" ref="product.product_template_only_form_view"/>
+            <field name="arch" type="xml">
+                    <xpath expr="//field[@name='barcode']" position="after">
+                        <field name="l10n_tr_gibp_number"
+                               invisible="product_variant_count &gt; 1
```

Comment:
Why not allowing to see/change it when multiple variants of the same products exist ? Wouldn't they most likely have the same number or not ?